website/manual.md

```diff
@@ -784,6 +784,43 @@ These Deno logos, like the Deno software, are distributed under the MIT license
 
 - [A pixelated animation one by @tanakaworld](https://github.com/denoland/deno/blob/master/website/images/deno_logo_4.gif)
 
+## Frequently Asked Questions
+
+### Is Deno production ready?
+
+No.
+
+### Will there be an official package manager for Deno?
+
+No.
+
+### How came the Deno name?
+
+Just swap the letters and `node` become `deno`.
```

Comment:
Remove this. It's obvious and not practical information. It doesn't need to be said.